src/bdev.rs

```diff
@@ -155,28 +162,122 @@ pub struct IoWaitEntry{
     wentry: spdk_bdev_io_wait_entry,
 }
 
+#[derive(Debug)]
+pub struct BdevIo{
+    ptr: *mut spdk_bdev_io,
+}
+
+impl BdevIo{
+    pub fn free_io(&self){
+        unsafe{
+            spdk_bdev_free_io(self.ptr)
+        };
+    }
+}
+
+#[derive(Debug)]
+pub struct dma_buf{
```

Comment:
We should follow Rust naming conventions and use `CamelCase` for type here.
```suggestion
pub struct DmaBuf {
```